models/zendesk__ticket_field_history.sql

```diff
@@ -25,7 +25,7 @@ with change_data as (
     select 
         *
     from {{ this }}
-    where date_day = (select max(date_day) from {{ this }} )
+    where date_day > (select max(date_day) from {{ this }} )
```

Comment:
As well as the above comment, we do not actually want to change this. We do want to focus in on the **most recent change** and therefore we would want to capture the latest change using `=` in this cte.